go/logger/standard.go

```diff
@@ -355,37 +355,37 @@ func (log *Standard) GetUnforwardedLogger() *UnforwardedLogger {
 }
 
 func (log *UnforwardedLogger) Debug(s string, args ...interface{}) {
-	if log.internal != nil {
+	if log != nil && log.internal != nil {
```

Comment:
hmm well that helps us track it down a little bit better. @songgao was going to look into it a little bit further.